base/number.jl

```diff
@@ -223,6 +223,7 @@ map(f, x::Number, ys::Number...) = f(x, ys...)
 
 """
     zero(x)
+    zero(T::type)
```

Comment:
I think it's a little clearer if this is labeled as `zero(::Type{<:Number})`

Reply:
There are types which are not `<:Number` but support `zero`, e.g. `Dates.Day`.

Reply:
True, but those are not hitting this particular method.  It's probably okay to just call it `::Type` then though.

Reply:
OK. I change it to `::Type`